src/FakeItEasy/Creation/IProxyOptions.cs

```diff
@@ -12,6 +12,6 @@ internal interface IProxyOptions
 
         IEnumerable<Action<object>> ProxyConfigurationActions { get; }
 
-        IEnumerable<CustomAttributeBuilder> AdditionalAttributes { get; }
+        IEnumerable<Expression<Func<Attribute>>> AdditionalAttributes { get; }

```

Comment:
I considered changing it, but it was consistent with `AdditionalInterfacesToImplement`... I'll rename it if you prefer.